autosklearn/automl.py

```diff
@@ -298,7 +299,8 @@ def _fit(self, datamanager, metric):
                                      "of " + str(allowed_elements))
         if self._resampling_strategy not in ['holdout', 'holdout-iterative-fit',
                                              'cv', 'partial-cv',
-                                             'partial-cv-iterative-fit']:
+                                             'partial-cv-iterative-fit'] and not isinstance(self._resampling_strategy,
+                                                                                           BaseCrossValidator):
```

Comment:
It would be good if you also check that no resampling strategy arguments are provided if the user provides a resampling strategy.

Reply:
Could you please reformat this a bit, for example like:
```
if self._resampling_strategy not in [
    'holdout', 
    'holdout-iterative-fit',
    'cv', 
    'partial-cv',
    'partial-cv-iterative-fit'
] and not isinstance(
    self._resampling_strategy, BaseCrossValidator
):
```
I know that there's no style guide, but it would good to stick to PEP8 as much as possible.